lib/chitter.rb

```diff
@@ -0,0 +1,49 @@
+require 'sinatra/base'
+require_relative 'data_mapper_setup'
+
+class Chitter < Sinatra::Base
+  enable :sessions
+
+  get '/' do
+    erb :index
+  end
+
+  get '/peeps/all' do
+  #   @peeps = Peep.all
+    erb :'peeps/all'
+  end
+
+  get '/user/new' do
+    erb :'user/new'
+  end
+
+  post '/user/new' do
+    user = User.new(email: params[:email],
+                    username: params[:username],
+                    password: params[:password],
+                    password_confirm: params[:password_confirm],
+                    name: params[:name])
+    if user.save
```

Comment:
Style/IfUnlessModifier: Favor modifier `if` usage when having a single-line body. Another good alternative is the usage of control flow `&&`/`||`.